src/Workspaces/Core/Portable/Formatting/Rules/BaseIndentationFormattingRule.cs

```diff
@@ -187,7 +187,7 @@ private static TextSpan GetSpanFromTokens(TextSpan span, SyntaxToken token1, Syn
             // current token is inside of the given span, get next token's start position.
             if (token2.Span.End <= span.End)
             {
-                token2 = token2.GetNextToken();
+                token2 = token2.GetNextToken(includeZeroWidth: true);
                 end = token2.Span.Start;
                 if (token2.RawKind == 0)
```

Comment:
this line is strange. and also line above.

I think it should be

var previous/nextToken = token1.GetPreviousToken()/GetNextToken();
...
if (preview/nextToken.RawKind == 0)
{
start/end = token1/2.FullSpan.Start/End;
}